db/migrate/20190214125845_add_comments_to_miq_tasks_table.rb

```diff
@@ -0,0 +1,21 @@
+class AddCommentsToMiqTasksTable < ActiveRecord::Migration[5.0]
+  def up
+    change_table_comment :miq_tasks, "ManageIQ background tasks"
+
+    change_column_comment :miq_tasks, :context_data, "Serialized metadata that may contain more detailed information about the task."
+    change_column_comment :miq_tasks, :created_on, "The timestamp the task was added to the app inventory."
+    change_column_comment :miq_tasks, :id, "The internal database ID."
+    change_column_comment :miq_tasks, :identifier, "The identifier used by metrics rollups for EMS clusters."
```

Comment:
I don't think it is intended to be only for metric rollups, just happens to be only used there.

More accurately might be described as "The resource this task is related to, format `"#{klass}:#{id}"`.  Honestly this would make more sense as a polymorphic resource than a string column that jams the class+id together.

Reply:
@bdunne @agrare Ok, I've updated it.